### PR TITLE
customisation and animation homework

### PR DESCRIPTION
**Pull request by Jinesra**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        android:layout_margin="@dimen/layout_margin"
+        android:contentDescription="@string/yellow_star"
+        app:srcCompat="@android:drawable/btn_star_big_on"
+        tools:ignore="ImageContrastCheck" />
+
+    <Button
+        android:id="@+id/button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/toast"
```
Comment (roman-koshulinskyy-ssw) [inline app/src/main/res/layout/activity_main.xml]: Missing resource.
Build fails locally.
Please add the missing string.

**Review by roman-koshulinskyy-ssw — APPROVED** [review]
The code has been reviewed and checked on a device locally.
In general looks acceptable providing that the build error is resolved.
The app is simple, but the idea is interesting :)

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Jinesra) [reply app/src/main/res/layout/activity_main.xml]: Hi! Sorry, I was wrong and sent the wrong homework. I will fix it tomorrow.

Comment (Jinesra) [reply app/src/main/res/layout/activity_main.xml]: My application was in a folder that is in the folder that I pushed to the repository.

**Comment by roman-koshulinskyy-ssw** [comment]
UPD: 
Checked the updated code. 
Looks acceptable.

**Pull request merged**